### PR TITLE
[AJ-693, AJ-750, AJ-752] TSV upload modal tweaks for Azure

### PR DESCRIPTION
**Pull request by davidangb**
tested by running locally.

reviewer: the first three commits each tackle one of the three Jira tickets AJ-693, AJ-750, AJ-752. The fourth commit handles Beth's PR feedback below; review commit-by-commit if that's helpful to you.

No changes for GCP workspaces.

For Azure workspaces:
* hide the `Terra Support: Importing Data - Using a Template` link, since the link points at GCP-specific instructions
* hide the `Click here for more info on the table` link, since it also points at GCP-specific instructions
* change the `Download sample_template.tsv` link to point at a file in an open-access Azure container instead of a GCP-hosted file
* change the placeholder text for the `Text Import` textarea to remove the `entity:` prefix on the first column

# GCP (remember, no changes here):
Screenshot 12-6-2022 at 04 19 PM
Screenshot 12-6-2022 at 04 42 PM

# Azure:
Screenshot 12-8-2022 at 02 42 PM
Screenshot 12-8-2022 at 02 42 PM (1)

### CODE REVIEW

File: src/libs/ajax/data-table-providers/WdsDataTableProvider.ts [hunk 1]
```diff
@@ -104,7 +104,8 @@ export class WdsDataTableProvider implements DataTableProvider {
 
   tsvFeatures: TSVFeatures = {
     needsTypeInput: true,
-    sampleTSVLink: 'https://storage.googleapis.com/terra-featured-workspaces/Table_templates/template_sample-wds-table.tsv', //TODO: This location may need to change
+    sampleTSVLink: 'https://azurefeaturedworkspace.blob.core.windows.net/featuredworkspacedata/template_data_table_Azure.txt',
+    dataImportSupportLink: '',
```
Comment (davidangb) [inline src/libs/ajax/data-table-providers/WdsDataTableProvider.ts:108]: if/when we have a WDS-specific support article, we can add its link here and then the UI will start showing the link again.

**Comment by bethsheets** [comment]
@davidangb can you remove the top text and link "choose... " as well? It points to documentation for entity service.

**Review by sagehen03 — APPROVED** [review]

**Comment by jordansafer** [comment]
A minor comment on the Azure page, the "Enter a table name" text box is a bit close to the File Import/Text Import selector, which makes the page feel cramped. I know this isn't part of this PR! I can make a separate ticket if that makes more sense-

**Review by jordansafer — APPROVED** [review]
LGMT!

**Comment by davidangb** [comment]
@jordansafer in cf9e457d74ad443f829ecc467064297b79fd7fce I added some more vertical spacing (and updated the screenshots). How does this look?

**Comment by jordansafer** [comment]
> @jordansafer in [cf9e457](https://github.com/DataBiosphere/terra-ui/commit/cf9e457d74ad443f829ecc467064297b79fd7fce) I added some more vertical spacing (and updated the screenshots). How does this look?

It looks great!

**Pull request merged**